x11-libs/wxgtk/wxgtk-3.1.5.recipe

```diff
@@ -0,0 +1,196 @@
+	devel:libwx_gtk3u_html_3.1$secondaryArchSuffix = $libVersionCompat
+	devel:libwx_gtk3u_propgrid_3.1$secondaryArchSuffix = $libVersionCompat
+	devel:libwx_gtk3u_qa_3.1$secondaryArchSuffix = $libVersionCompat
+	devel:libwx_gtk3u_ribbon_3.1$secondaryArchSuffix = $libVersionCompat
+	devel:libwx_gtk3u_richtext_3.1$secondaryArchSuffix = $libVersionCompat
+	devel:libwx_gtk3u_stc_3.1$secondaryArchSuffix = $libVersionCompat
+	devel:libwx_gtk3u_xrc_3.1$secondaryArchSuffix = $libVersionCompat
+	"
+
+REQUIRES_devel="
+	wxgtk$secondaryArchSuffix == $portVersion base
+	xcairo${secondaryArchSuffix}_devel
+	devel:libgdk_3$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libticonv$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	xcairo${secondaryArchSuffix}_devel
+	devel:libatk_1.0$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libfontconfig$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgdk_3$secondaryArchSuffix
+	devel:libgdk_pixbuf_2.0$secondaryArchSuffix
+	devel:libgtk_3$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	devel:libpango_1.0$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libSM$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libX11$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_PREREQUIRES="
+	cmd:find
+	cmd:gawk
+	cmd:gcc$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage wxgtk$secondaryArchSuffix \
+	"$libDir"/libwx_baseu-3.1.so.$libVersion \
+	"$libDir"/libwx_baseu_net-3.1.so.$libVersion \
+	"$libDir"/libwx_baseu_xml-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_adv-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_aui-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_core-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_html-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_propgrid-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_qa-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_ribbon-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_richtext-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_stc-3.1.so.$libVersion \
+	"$libDir"/libwx_gtk3u_xrc-3.1.so.$libVersion
+
+BUILD()
+{
+	runConfigure ./configure \
+	--with-cxx=11 \
+	--with-gtk \
+	--enable-shared \
+	--enable-printfposparam \
+	--enable-unicode \
+	--enable-sound \
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	prepareInstalledDevelLibs \
+	libwx_baseu-3.1 \
```

Comment:
Same here.

Reply:
Hopefully i understood you well? thanks for checking again.